how-to.md

```diff
@@ -121,3 +121,11 @@ You can verify the licenses in your tenant [in the Azure Portal](https://portal.
   * **Tenants** > **Repos** > select dropdown at the top of the page > **Manage repositories** > **...** (next to the tenant you want to remove) > **Delete** > follow the on-screen instructions to delete the repository
   * **Pipelines** > **...** (next to the tenant you want to remove) > follow the on-screen instructions to delete the pipeline
   * **Environments** (on the left pane under **Pipelines**) > select the tenant you want to remove > **...** > **Delete** > **Delete**
+
+## Add users to Simeon
+
+* Navigate to [DevOps](https://dev.azure.com/) > **Organization settings** > **Users** > **Add users** > enter the email(s) of the user(s) you are inviting > **Add**
```

Comment:
Do sub bullets to avoid repeating "navigate to", such as how we do it here https://simeoncloud.github.io/docs/#/how-to?id=set-up-billing-for-your-organization

Reply:
resolved